src/scaffold/emitter.rs

```diff
@@ -156,7 +156,7 @@ impl<'s> Visitor for EmitterI<'s> {
 
     fn visit_root(&mut self, root: &hir::Root) -> result::Result<Self::Output, Self::Error> {
         let mut emitted = String::new();
-
+        emitted.push_str(&format!("// SPDX-License-Identifier: UNLICENSED\n\n").as_str());
```

Comment:
We can remove the line between the license and the pragma. This means updating all the tests again.

```suggestion
        emitted.push_str(&format!("// SPDX-License-Identifier: UNLICENSED\n").as_str());
```

Reply:
Updated here: https://github.com/alexfertel/bulloak/pull/47/commits/7489421b53c2a49ddce36570f8b73cbfb0a2a448